AK/Math.h

```diff
@@ -9,16 +9,13 @@
 #include <AK/Concepts.h>
 #include <AK/StdLibExtraDetails.h>
 #include <AK/Types.h>
+#include <LibM/math_defines.h>
+#include <fenv.h>
 
+// FIXME: Some of these can be implemented using SSE, which might be faster on
```

Comment:
Maybe leave using sse to the compiler, unless shown important to do by hand? And I think if we do get to optimizing some of this by hand, we'll see that as an option without these comments.

Reply:
the compiler wont generate proper sse code if I handwrite x87 asm as the implementation